admin/superuser-init.py

```diff
@@ -60,95 +60,96 @@ def get_input_or_data(name):
-
-print "writing %s/python/config.py" % directory
-with open(directory + '/python/config.py', 'w') as new_configfile, open(utils.get_path('./conference-template/python/config.template', __file__)) as template_configfile:
-    templatestring = template_configfile.read().replace('<<CONF>>', "\"%s\"" % conference)
-    templatestring = templatestring.replace('<<SUBMISSION_NAME>>',submission_name)
-    templatestring = templatestring.replace('<<TIMESTAMP>>',str(duedate_milliseconds))
-    new_configfile.write(templatestring)
-
-print "writing %s/webfield/conf.html" % directory
-with open(directory + '/webfield/conf.html', 'w') as new_webfile, open(utils.get_path('./conference-template/webfield/conf.template',__file__)) as template_webfile:
-    templatestring = template_webfile.read().replace('<<TITLE>>',"\"%s\"" % conference_title)
-    templatestring = templatestring.replace('<<CONF>>',"\"%s\"" % conference)
-    templatestring = templatestring.replace('<<SUBMISSION_NAME>>', submission_name)
-    templatestring = templatestring.replace('<<SUBTITLE>>',"\"%s\"" % conference_subtitle)
-    templatestring = templatestring.replace('<<LOCATION>>',"\"%s\"" % conference_location)
-    templatestring = templatestring.replace('<<CONF_DATE>>',"\"%s\"" % conference_date)
-    templatestring = templatestring.replace('<<DATE>>',"%s" % human_duedate)
-    templatestring = templatestring.replace('<<URL>>',"\"%s\"" % url)
-    new_webfile.write(templatestring)
-
-print "writing %s/python/admin-init.py" % directory
-with open(directory + '/python/admin-init.py', 'w') as new_initfile, open(utils.get_path('./conference-template/python/admin-init.template', __file__)) as template_initfile:
-    templatestring = template_initfile.read().replace('<<SUBMISSION_DUEDATE>>', "%s" % duedate_milliseconds)
-    new_initfile.write(templatestring)
-
-print "writing %s/process/submissionProcess.template" % directory
-with open(directory + '/process/submissionProcess.template', 'w') as new_submissionprocess, open(utils.get_path('./conference-template/process/submissionProcess.template', __file__)) as template_submissionprocess:
-    templatestring = template_submissionprocess.read().replace('<<CONF>>', "\"%s\"" % conference)
-    templatestring = templatestring.replace('<<PHRASE>>', "\"%s\"" % conference_phrase)
-    new_submissionprocess.write(templatestring)
-
-print "writing %s/process/commentProcess.js" % directory
-with open(directory + '/process/commentProcess.js', 'w') as new_commentprocess, open(utils.get_path('./conference-template/process/commentProcess.template', __file__)) as template_commentprocess:
-    templatestring = template_commentprocess.read().replace('<<PHRASE>>', "\"%s\"" % conference_phrase)
-    new_commentprocess.write(templatestring)
-
-print "writing %s/params.json" % directory
-with open(directory + '/params.json', 'w') as paramsfile:
-    json.dump(data, paramsfile, indent=4)
+
+if args.overwrite or not os.path.exists(directory):
```

Comment:
sorry, the "suggestion" isn't a "default" - you would still have to enter the username you want. I can change this.